emacs/merlin.el

```diff
@@ -412,12 +433,12 @@ position POS."
     (merlin--goto-point data)
     (point)))
 
-(defun merlin/unmake-point (point)
-  "Destruct POINT to line / col."
-  (save-excursion
-    (goto-char point)
-    (beginning-of-line)
-    (format "%d:%d" (line-number-at-pos) (- point (point)))))
+;; Accept extra parameters to support merlin-eldoc which relies on this API.
```

Comment:
Not sure what `merlin-eldoc` is, but aren't you breaking it if you just ignore the argument?

Reply:
Ah sorry, I had missed the fact that @Khady's aware of this.
Then I'd say we should just remove the argument, instead of ignoring it.